.drone.yml

```diff
@@ -14,7 +14,7 @@ resources:
 
 steps:
   - name: test
-    image: golangci/golangci-lint:v1.56.0
+    image: golangci/golangci-lint:v2.5.0
```

Comment:
required due to Go 1.25 upgrade, I'm also keeping this in sync with 10gen/platform and 10gen/kanopy repos